modules/ROOT/pages/cache.adoc

```diff
@@ -0,0 +1,49 @@
+= Optionally Configure the Platform Cache
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+You can improve the performance of pages and components in your API community by using the https://help.salesforce.com/articleView?id=data_platform_cache.htm&type=5[Platform Cache, role=external, window=blank].
```

Comment:
```suggestion
You can improve the performance of pages and components in your API community by using the https://help.salesforce.com/articleView?id=data_platform_cache.htm&type=5[platform cache, role=external, window=blank].
```

Reply:
agree as long as we make sure lower-case for "platform cache" is used consistently across docs